### PR TITLE
Fluss+ Branding for integration in Home-Assistant Core

### PR DESCRIPTION
**Pull request by NjeruFluss**
## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #
- Link to code base pull request: https://github.com/home-assistant/core/pull/121599
- Link to documentation pull request:https://github.com/home-assistant/home-assistant.io/pull/33801
- Link to integration documentation on our website: 
- Link to custom integration repository: 

## Checklist

- [x] The added/replaced image(s) are **PNG**
- [x] Icon image size is 256x256px (`icon.png`)
- [x] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [x] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [x] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

**Review by frenck — CHANGES_REQUESTED** [review]
There is a CI error, can you please take a look?

../Frenck

**Comment by NjeruFluss** [comment]
Hi @frenck, I already have the Fluss branding as a custom_integration. Is it necessary to change the domain associated with Fluss to get it pulled into the core_integrations? Or is there another way of keeping the domain the same on both?

**Comment by frenck** [comment]
If you want to keep a custom integration, I recommend on changing the domain of the custom integration to prevent collision with the core integration you've been submitting.

If you are migrating to core fully, the custom integration one can be removed in this PR.

File: custom_integrations/fluss_pluss/icon.png
Comment (frenck) [inline custom_integrations/fluss_pluss/icon.png:1]: This is a rename of the custom integration, which should be done in a separate PR.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — CHANGES_REQUESTED** [review]
Seems like something went wrong, as this PR is now an empty pull request:

CleanShot 2024-07-16 at 08 51 15@2x


Can you take a look?

../Frenck

**Comment by frenck** [comment]
You made this comment on the documentation, which should probably have been placed on this PR

CleanShot 2024-07-17 at 08 52 36@2x

As for that comment, @NjeruFluss; That is not correct.

There is no `fluss` domain in the `core_integrations` in this repository, which is what this PR should be adding.

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**